entity/NetSuiteConfigEntity.xml

```diff
@@ -42,4 +42,36 @@ along with this software (see the LICENSE.md file). If not, see
             <key-map field-name="systemMessageRemoteId"/>
         </relationship>
     </entity>
+
+    <entity entity-name="LoopReturnHistory" package="co.hotwax.netsuite" use="configuration">
+        <field name="loopReturnId" type="id" is-pk="true"/>
+        <field name="partyId" type="id" is-pk="true"/>
+        <field name="systemMessageRemoteId" type="id"/>
+        <field name="shopifyOrderId" type="id"/>
+        <field name="shopifyOrderName" type="id"/>
+        <field name="netsuiteReturnId" type="id"/>
+        <field name="statusId" type="id"/>
+        <field name="createdDate" type="date-time"/>
+        <field name="lastModifiedDate" type="date-time"/>
```

Comment:
I don;t think we need to keep the lastModifiedDate, as moqui internally append the lastModifiedStamp in database table